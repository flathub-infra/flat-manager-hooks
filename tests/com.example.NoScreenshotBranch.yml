id: com.example.NoScreenshotBranch
runtime: org.freedesktop.Platform
sdk: org.freedesktop.Sdk
runtime-version: '22.08'
command: main
build-options:
  env:
    APP_ID: com.example.NoScreenshotBranch
modules:
- name: build
  buildsystem: simple
  build-commands:
  - gcc -o main main.c
  - install -D main /app/bin/main
  - sed -i "s/@APP_ID@/$APP_ID/g" appstream.xml
  - install -D appstream.xml /app/share/metainfo/$APP_ID.appdata.xml
  - sed -i "s/@APP_ID@/$APP_ID/g" app.desktop
  - install -D app.desktop /app/share/applications/$APP_ID.desktop
  - install -D app_icon.svg /app/share/icons/hicolor/scalable/apps/$APP_ID.svg
  sources:
  - type: file
    path: main.c
  - type: file
    path: appstream.xml
  - type: file
    path: app_icon.svg
  - type: file
    path: app.desktop
