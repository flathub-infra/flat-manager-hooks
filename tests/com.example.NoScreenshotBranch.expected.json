{
  "diagnostics": [
    {
      "refstring": "app/com.example.NoScreenshotBranch/x86_64/master",
      "is_warning": false,
      "info": {
        "category": "NoScreenshotBranch"
      }
    }
  ]
}
